charts/portworx/templates/hooks/post-install/px-create-cluster-token.yaml

```diff
@@ -0,0 +1,125 @@
+{{- if (.Values.clusterToken.create) }}
+  {{- $customRegistryURL := .Values.customRegistryURL | default "none" }}
+  {{- $registrySecret := .Values.registrySecret | default "none" }}
+
+apiVersion: batch/v1
+kind: Job
+metadata:
+  namespace: kube-system
+  name: px-set-cluster-token
+  labels:
+    heritage: {{.Release.Service | quote }}
+    release: {{.Release.Name | quote }}
+    chart: "{{.Chart.Name}}-{{.Chart.Version}}"
+    app.kubernetes.io/managed-by: {{.Release.Service | quote }}
+    app.kubernetes.io/instance: {{.Release.Name | quote }}
+  annotations:
+    "helm.sh/hook": post-install
+    "helm.sh/hook-weight": "10"
+    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
+spec:
+  backoffLimit: 0
+  template:
+    spec:
+      {{- if not (eq $registrySecret "none") }}
+      imagePullSecrets:
+        - name: {{ $registrySecret }}
+      {{- end }}
+      restartPolicy: Never
+      serviceAccountName: {{ .Values.clusterToken.serviceAccountName }}
+      containers:
+        - name: post-install-job
+          {{- if eq $customRegistryURL "none" }}
+          image: "bitnami/kubectl:{{ template "px.kubectlImageTag" . }}"
+          {{- else}}
+          image: "{{ $customRegistryURL }}/bitnami/kubectl:{{ template "px.kubectlImageTag" . }}"
+          {{- end }}
+          env:
+            - name: NS
+              value: kube-system
+            - name: KEY
+              value: cluster-wide-secret-key
+          command: ['/bin/bash', '-c']
+          args:
+            - kubectl -n $NS exec -c portworx $(kubectl -n $NS get pods -lname=portworx -o jsonpath='{range .items[*]}{.status.conditions[?(@.type=="Ready")].status}{" "}{.metadata.name}{"\n"}{end}' | grep True | cut -d' ' -f2 | head -n1) -- /opt/pwx/bin/pxctl secrets set-cluster-key --secret $KEY
```

Comment:
I could not think of any simpler command to do this and the init container. If you have suggestions let me know.

Reply:
yes that's hard to read and understand.. are we able to put it in multiple lines or add comments?

Reply:
Added multiple lines and some comments. Let me know if this is more readable. If not, I can update it in a subsequent PR.